build.gradle

```diff
@@ -33,17 +32,14 @@ loom {
 dependencies {
 	// To change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	mappings "net.fabricmc:yarn:${project.yarn_version}:v2"
```

Comment:
This no longer matches the develop page for versioning 

https://fabricmc.net/develop/

Reply:
Yeah they'll have to be updated there too if this is merged